### DISCUSSION TITLE
can not install on Delphi 10.4 Community Edition

### DISCUSSION THREAD

**Discussion by divinity76** [project-jedi/jcl#127] [General]
trying to install jcl on "Delphi 10.4 Community Edition" will just generate this error from dcc32.exe:
```
C:\cygwin64\home\hans\projects\jcl\jcl>Install.bat
Initializing/Updating git submodules...

===================================================================
Compiling JediIncCheck...
Using Embarcadero RAD Studio 10.4

This version of the product does not support command line compiling.


An error occured while compiling the installer. Installation aborted.

Press any key to continue . . .

C:\cygwin64\home\hans\projects\jcl\jcl>
```
yeah.. 10.4 Community comes with a crippled dcc32.exe it seems :/  any idea how to install it on 10.4 Community?

**Comment by divinity76** [comment]
did some digging: per https://blogs.embarcadero.com/delphi-cbuilder-community-editions-now-available-in-version-10-4-2/#comment-1339 . Community Edition 10.4 removed support for command-line compilation... any idea where to get Community Edition 10.3?

**Comment by obones** [comment]
For the community edition, the only solution is to use the "Manual Installation" as described in this document:
https://github.com/project-jedi/jcl/blob/master/jcl/docs/Readme.txt
